docs/agent-api.asciidoc

```diff
@@ -60,6 +60,14 @@ Use `isStarted()` to check if the agent has already started.
 Returns `true` if the agent has started,
 otherwise returns `false`.
 
+[[apm-get-service-name]]
+==== `apm.getServiceName()`
+
+[small]#Added in: v3.11.0#
```

Comment:
+1 -- hadn't realized we marked all our new API methods with an _Added in_ until I saw this here.